### PR TITLE
Restore loading env from config file

### PR DESCRIPTION
**Pull request by Itxaka**
Useful to pass and set env vars during install for things like the docker config location

### CODE REVIEW

**Comment by Itxaka** [comment]
This was probably lost during the converge of the cloud config into config and such.

File: internal/agent/install.go [hunk 1]
```diff
@@ -226,6 +226,9 @@ func Install(dir ...string) error {
 }
 
 func RunInstall(c *config.Config) error {
+	utils.SetEnv(c.Env)
+	utils.SetEnv(c.Install.Env)
```
Comment (mauromorales) [inline internal/agent/install.go:230]: do these add on top of the other?

Comment (mauromorales) [reply internal/agent/install.go:230]: Gotcha, thanks! And yeah I guess long term having per action might be a good idea just like you have flags per action

**Review by mauromorales — APPROVED** [review]
LGTM

**Pull request merged**